src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,176 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static class Node<T> {
+        T item;
```

Comment:
add access modifier and move this class to the bottom of file please

Reply:
Ok, done